internal/cli/runner_profile_set.go

```diff
@@ -124,7 +129,11 @@ func (c *RunnerProfileSetCommand) Run(args []string) int {
 				},
 			},
 		}
-	} else {
+		if c.flagTargetRunnerAny != nil {
+			c.ui.Output("Both -target-runner-id and -target-runner-any detected, only one can be set at a time. ID takes priority.",
```

Comment:
I think that this is meant to be `Both -target-runner-label and -target-runner-any detected...`

```terminal
$ ./waypoint runner profile set -name=docker-def -target-runner-label=env=production -target-runner-any -default=false
✓ Runner profile updated
Both -target-runner-id and -target-runner-any detected, only one can be set at a time. ID takes priority.
```

Reply:
oops, copy paste error on my part! Good catch